src/map_object.c

```diff
@@ -933,22 +896,22 @@ static void sub_02062604(MapObject *mapObj)
 
 static void sub_02062618(MapObject *mapObj)
 {
-    MapObject_SetStatusFlagOff(mapObj, MAP_OBJ_STATUS_15 | MAP_OBJ_STATUS_HIDE_SHADOW | MAP_OBJ_STATUS_26 | MAP_OBJ_STATUS_24);
+    MapObject_SetStatusFlagOff(mapObj, MAP_OBJ_STATUS_SHOW_SHADOW | MAP_OBJ_STATUS_HIDE_SHADOW | MAP_OBJ_STATUS_26 | MAP_OBJ_STATUS_24);
 }
 
 static void sub_02062628(MapObject *mapObj)
 {
     (void)0;
 }
 
-static int sub_0206262C(FieldSystem *fieldSystem, int param1)
+static int FieldSystem_GetGraphicsID(FieldSystem *fieldSystem, int graphicsID)
```

Comment:
Revert renaming this function for now. There's more that meets the eye in `sub_0203F164`